dist/index.html

```diff
@@ -0,0 +1,106 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="/style.e308ff8e.css">
+  <script src="/style.e308ff8e.js"></script></head>
+  <body>
+    <form class="form">
+      <fieldset class="form__fieldsets">
+        <legend class="form__legends">
+          Personal information
+        </legend>
+        <label class="form__label-data">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+        <label class="form__label-data">
+          Name:
+          <input type="text" name="name" autocomplete="off">
+        </label>
+        <label class="form__label-data">
+          How old are You?
+          <input type="number" name="age" min="1" max="100" value="12">
+        </label>
+        <label class="form__label-data">
+          Full date of birth:
+          <input type="date" name="fullAge">
+        </label>
+        <label class="form__label-accept">
+          I accept the terp of the agreement
+          <input type="checkbox" name="terms" required="">
+        </label>
+      </fieldset>
+      
+      <fieldset class="form__fieldsets">
+        <legend class="form__legends">
+          Registration
+        </legend>
+        <label class="form__label-email">
+          E-mail:
+          <input required="" type="email" name="email" placeholder="email@example.com">
```

Comment:
check this for whole project
```suggestion
          <input required type="email" name="email" placeholder="email@example.com">
```

Reply:
ready